src/BubbleSort-Ten-Numbers.c

```diff
@@ -1,28 +1,66 @@
+/*
+
+Module: BubbleSort-Ten-Numbers.c
+
+Function:
+    Bubble sorting given 10 numbers from user and print in ascending order.
```

Comment:
use correct grammar - ```Bubble sort given numbers....```

Reply:
Changed those spelling as you mentioned.